lib-cloudsdk/src/main/java/com/facebook/openwifi/cloudsdk/models/ap/Capabilities.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+package com.facebook.openwifi.cloudsdk.models.ap;
+
+import java.util.Map;
+
+import com.google.gson.annotations.SerializedName;
+
+public class Capabilities {
+	public String compatible;
+	public String model;
+	public String platform;
+	public Map<String, String[]> network;
+
+	public static class Switch {
+		public boolean enable;
+		public boolean reset;
+	}
+
+	@SerializedName("switch") public Map<String, Switch> switch_;
+
+	public static class Phy {
```

Comment:
You're missing a few fields:
- `country` (string)
- `dfs_region` (integer)
- `temperature` (integer)

Sources:
- https://github.com/blogic/ucentral-wifi/blob/aa9c3ee716929d75852a11640362a09bb32c5262/nl80211.c#L787
- https://github.com/blogic/ucentral-wifi/blob/aa9c3ee716929d75852a11640362a09bb32c5262/nl80211.h#L15

Reply:
oh wow good to know - i was looking at https://github.com/Telecominfraproject/wlan-ucentral-schema/blob/bfcb0e8c6e6c07195ba964f696c562aae3723fa3/tests/mocks/ubus/wifi%7Ephy.json